libs/lawnchair/lawnchair.js

```diff
@@ -162,4 +162,12 @@ Lawnchair.prototype = {
     return this
   }
 // --
-};
\ No newline at end of file
+};
+
+require('./lawnchairHtml5FileSystem')(Lawnchair)
```

Comment:
Instead of using concat, these are loaded via `require` now so browserify will pick them up.